blueocean-pipeline-api-impl/src/main/java/io/jenkins/blueocean/listeners/BluePipelineAction.java

```diff
@@ -1,4 +1,4 @@
-package io.jenkins.blueocean.service.embedded.rest;
+package io.jenkins.blueocean.listeners;
```

Comment:
We should really move these over inside `io.jenkins.blueocean.rest.impl.pipeline`. These are strictly implementation specific and for the purpose of node API implementation so should be put inside `io.jenkins.blueocean.rest.impl.pipeline.listener`?